packages/react-theme/package.json

```diff
@@ -18,11 +18,11 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "echo \"This is DEPRECATED, use 'storybook' instead\" && just-scripts dev:storybook",
+    "start": "storybook",
     "storybook": "start-storybook",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "test": "jest",
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/packages/react-theme/src && yarn docs"
```

Comment:
This fails because `dist/packages/react-theme/src` does not exist. Who is supposed to create the directory?
![image](https://user-images.githubusercontent.com/9615899/122909603-a642ab00-d355-11eb-9a75-d9705b3d23d2.png)

Reply:
nice catch! looks like TS changed outDir mechanism, previously running `tsc -p . --module esnext --emitDeclarationOnly` would mirror the structure, thus creating 

`dist/packages/<package-name>`

Looks like that `packages` folder needs to be removed from the `--output` option -> fix needed for migration generator

Reply:
https://github.com/microsoft/fluentui/issues/18659

Reply:
path fixed